shopfloor_mobile/static/wms/src/scenario/zone_picking.js

```diff
@@ -606,7 +606,19 @@ const ZonePicking = {
                 select_line: {
                     display_info: {
                         title: "Select move",
-                        scan_placeholder: "Scan location / pack / product / lot",
+                        scan_placeholder: () => {
+                            const sublocation = this.state.data.sublocation;
+                            if (
+                                this.state.data.scan_location_or_pack_first &&
+                                !sublocation
+                            ) {
+                                return "Scan location / pack 000";
```

Comment:
```suggestion
                                return "Scan location / pack";
```

Reply:
Oops, left over from my testing.
I fixed that. And squashed.

Reply:
And rebased